source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -2567,6 +2599,10 @@ export class HTML5MediaPlayer extends MediaPlayer {
                 }
             }
         }
+
+        for (const captionSource of this.owner.captionSources) {
+            this._captionSources.push(captionSource);
+        }
```

Comment:
```suggestion
        this._captionSources.push(...captionSource);
```

Reply:
_what is this sorcery_

Reply:
> _what is this sorcery_

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax

Reply:
haha thanks yeah I looked it up too, was mainly expressing my dumbfoundedness that this is a thing

Reply:
I'm responsible for more occurrences of this type of `for` loop in the codebase... maybe at some point we could change them all to using the spread operator. For that matter I wonder if it has a performance benefit.